azext_edge/constants.py

```diff
@@ -7,7 +7,7 @@
 
 import os
 
-VERSION = "1.2.0a1"
+VERSION = "1.2.0b3"
```

Comment:
going to assume this is the correct version and/or will be fixed later

Reply:
yea will be fixed right after this merge